drivers/mbox/mbox_stm32_hsem.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2024 Celina Sophie Kalus <hello@celinakalus.de>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/device.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/mbox.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+
+#include "stm32_hsem.h"
+
+#define LOG_LEVEL CONFIG_MBOX_LOG_LEVEL
+#include <zephyr/logging/log.h>
+#include <zephyr/irq.h>
+LOG_MODULE_REGISTER(mbox_stm32_hsem_ipc);
```

Comment:
#include  should be defined  in alphabetical order and you could use LOG_MODULE_REGISTER

```c
#include <zephyr/device.h>
#include <zephyr/drivers/clock_control.h>
#include <zephyr/drivers/clock_control/stm32_clock_control.h>
#include <zephyr/drivers/mbox.h>
#include <zephyr/irq.h>
#include <zephyr/logging/log.h>


#include "stm32_hsem.h"

LOG_MODULE_REGISTER(mbox_stm32_hsem_ipc, CONFIG_MBOX_LOG_LEVEL);
```

Reply:
Thank you; I see I've been looking at the IPM driver too long to realize what should be done better. Fixed it with force push.